composer.json

```diff
@@ -385,5 +385,6 @@
             "Magento\\PhpStan\\": "dev/tests/static/framework/Magento/PhpStan/"
         }
     },
-    "prefer-stable": true
+    "prefer-stable": true,
+    "minimum-stability": "dev"
```

Comment:
I don't think the minimum stability should be dropped to dev. Is that required for some reason?